test/e2e/framework/util.go

```diff
@@ -3782,22 +3782,31 @@ type SSHResult struct {
 // is no error performing the SSH, the stdout, stderr, and exit code are
 // returned.
 func SSH(cmd, host, provider string) (SSHResult, error) {
+	var (
+		stdout string
+		stderr string
+		code   int
+		err    error
+	)
 	result := SSHResult{Host: host, Cmd: cmd}
+	if provider == "vsphere" {
```

Comment:
Wanted to use password based authentication so that our automation setup for test execution does not require to deal with ssh-keygen and updating .ssh/authorized_keys in all nodes.

but GetSigner looks good. Let me reuse GetSigner -> case "local" for vsphere cloud provider. 
we will update local script.